### DISCUSSION TITLE
Is it possible get Tab Bars height on iOS?

### DISCUSSION THREAD

**Discussion by phuongwd** [jpudysz/react-native-unistyles#222] [Q&A] [answered]
Thanks for great library, 
I have question related to the ~bottom bar~ Tab Bars height on iOS.

Is it possible get ~bottom bar~ Tab Bars height on iOS?

Edited: ~bottom bar~ => Tab Bars

**Comment by jpudysz** [comment]
Hey 👋 

Yes, of course!

UnistylesRuntime.insets.bottom 

It's cross platform

**Reply by phuongwd** [reply]
@jpudysz thanks for response.

I mean the tab bars https://developer.apple.com/design/human-interface-guidelines/tab-bars
Causing, its return 0 on iPhone 8+, iPhone 15 Pro and iPhone 15 Pro Max.

**Comment by jpudysz** [comment, marked as answer]
Tab bars is your app content. 
You shold check the library documentation that you are using.
